### DISCUSSION TITLE
get upcoming ID

### DISCUSSION THREAD

**Discussion by IskanderMustafin** [typegoose/auto-increment#27] [Q&A]
Hello,

Is it possible somehow to get next/upcoming ID?
AFAIK `mongoose-auto-increment` decorated all models with `nextCount` method, which returned next ID. Is there some way to have similar functionality in `@typegoose/auto-increment`?   

Thanks

**Comment by hasezoey** [comment]
no, it currently is not possible to get the next count without incrementing it. for reference, which plugin function are you wanting this feature for? (`AutoIncrementSimple` or `AutoIncrementID`)

**Reply by IskanderMustafin** [reply]
Thanks for reply. I wanted it for `AutoIncrementID`. Sometimes it's useful to know incoming ID. In my case I need it for creating `AWS S3` folder for uploading there profile's picture before creating Profile entity itself.
